config/prod/bwhite-beat-aml-bioc-again.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: bwhite-beat-aml-bioc-again
+sceptre_user_data:
+  Distro: "aws"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  # OpenPorts: ["80","443"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "beat-aml"
+  # The resource owner
+  OwnerEmail: "brian.white@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  AMIId: "ami-08333c98cd3b5ada7"      
```

Comment:
I think you have a trailing white space on this line. There can't be any trailing white spaces.